### PR TITLE
add naip 2016and data request image, and alter discover service names

### PR DESCRIPTION
**Pull request by rkelson**
Pull request description
------------------------

### CODE REVIEW

**Comment by steveoh** [comment]
Is this ready for a review and publish or what? Your description is empty.

**Comment by rkelson** [comment]
appologies, I will review it and let you know if it is ready to merge.
the next one i will send will need review with a push tomorrow

Rick Kelson
Utah AGRC
801.538.3237
gis.utah.gov

**Comment by steveoh** [comment]
So this is the one you would like @stdavis to merge first thing in the morning?

**Comment by rkelson** [comment]
Nope, this PR is just page edits. After this PR is merged I will refork and
commit some posts to be merged in the morning

Rick Kelson
Utah AGRC
801.538.3237
gis.utah.gov

**Comment by rkelson** [comment]
ready to merge

File: data/sgid-base-map-services-arcmap/index.md [hunk 1]
```diff
@@ -27,18 +27,20 @@ Instructions for using Discover services in ArcMap 10.x:
...
-  - `hro_2012_rgb` – 12.5-cm natural color aerial photography collected in 2012 of the Wasatch Front
-  - `naip_*year*_rgb` – 1-meter statewide natural color aerial photography
-  - `naip_*year*_nrg` – 1-meter statewide color infrared aerial photography
-  - `hillshade` – 10 meter 4-way statewide shaded relief
-  - `address_points-basemap` – Base map of address point locations
-  - `lite-basemap` – Base map of hillshade, HWYs/streets, cities, boundaries, etc. (gray, muted tone)
-  - `overlay-basemap` – Base map of vector features HWYs/streets, cities, address points, parcels, etc. (transparent background)
-  - `terrain-basemap` – Base map of hillshade, HWYS/streets, parks, forests, water, etc.
-  - `topo-basemap` – Base map of USGS topographic maps
+  - `Utah` – Combination of the most recent 1-meter NAIP and 6-inch Google imagery with scale dependent rendering **(licensed)**
```
Comment (steveoh) [inline data/sgid-base-map-services-arcmap/index.md]: do these **'s work or are they interpreted as markdown?

Comment (rkelson) [inline data/sgid-base-map-services-arcmap/index.md]: they are bold so interpreted by MD

**Comment by rkelson** [comment]
I've been using them for a while:
https://gis.utah.gov/data/sgid-base-map-services-arcmap/

Rick Kelson
Utah AGRC
801.538.3237
gis.utah.gov

**Comment by steveoh** [comment]
The way the diff was word wrapped it looked funny. My mistake.

**Pull request merged**